law_tools/dashboard/torsion_map.py

```
@@ -0,0 +1,66 @@
+    """Return torsion norm values across polar coordinates."""
+
+    r = np.linspace(1, 10, size)
+    theta = np.linspace(0, 2 * np.pi, size)
+    R, TH = np.meshgrid(r, theta)
+    golden = (1 + np.sqrt(5.0)) / 2.0
+    base_phase = (2 * np.pi / 3.0) * R + perturb * golden
+    torsion = np.abs(np.sin(base_phase)) * chi_int
+    return TorsionMapResult(r=R, theta=TH, torsion=torsion)
+
+
+def make_torsion_heatmap(result: TorsionMapResult, chaos_threshold: float = 0.5):  # pragma: no cover
```

Ruff (0.13.3)

Unused function argument: `chaos_threshold`

(ARG001)